packages/types/src/base.ts

```diff
@@ -48,6 +48,8 @@ export type UtxoChainId =
   | KnownChainIds.DogecoinMainnet
   | KnownChainIds.LitecoinMainnet
 
+export type SvmChainId = KnownChainIds.SolanaMainnet
```

Comment:
preferably-blocking: remove me - this makes sense for EVM ChainIds as an intersection of EVM ChainIds, however, for Solana, SolanaMainnet is the only supported so we should either use `KnownChainIds.SolanaMainnet` directly (preferred for the sake of simplicity), or, if we really want to early abstract this for different SVM ChainIds, than this should be `SolanaChainId` as an alias for the time being